### PR TITLE
Added new/missing microbenchmarks for Aggregates,Joins and Orderby

### PR DESCRIPTION
**Pull request by shantanugupta-yb**
Added new/missing microbenchmarks for Aggregates,Joins and Orderby

### CODE REVIEW

File: config/yugabyte/regression_pipelines/aggregate_workloads/postgres/AGGRG10_aggregate_cardinality.yaml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                    - name: col_varchar500_id_
+                      count: 1
+                      util: PrimaryStringGen
+                      params: [1, 500]
+
+        executeRules:
+            - workload: AGGRG10_1_count_point_select_multiple_values
+              run:
+                  - name: count_bigint_non_indexed_column
+                    customTags: schematype=colocated,pkey=composite,partition=hash,aggregate=count,cardinality=multiple,projection=aggregate,aggregateon=rangepkey,queryshape=aggregate,filteron=rangepkey,filtertype=pointselect
```
Comment (sonalsagarwal) [inline config/yugabyte/regression_pipelines/aggregate_workloads/postgres/AGGRG10_aggregate_cardinality.yaml:77]: Do we add customTags for postgres?

File: config/yugabyte/regression_pipelines/aggregate_workloads/yb_colocated/AGGRG10_aggregate_cardinality.yaml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                    - name: col_varchar500_id_
+                      count: 1
+                      util: PrimaryStringGen
+                      params: [1, 500]
+
+        executeRules:
+            - workload: AGGRG10_1_count_point_select_multiple_values
+              run:
+                  - name: count_bigint_non_indexed_column
+                    customTags: schematype=colocated,pkey=composite,partition=hash,aggregate=count,cardinality=multiple,projection=aggregate,aggregateon=rangepkey,queryshape=aggregate,filteron=rangepkey,filtertype=pointselect
```
Comment (sonalsagarwal) [inline config/yugabyte/regression_pipelines/aggregate_workloads/yb_colocated/AGGRG10_aggregate_cardinality.yaml:79]: partition should be range

**Review by sonalsagarwal — CHANGES_REQUESTED** [review]
Please check the review comments

File: config/yugabyte/regression_pipelines/join_workloads/yb_colocated/JOING9_cascadded_joins.yaml [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+                  - name: join_on_pk_3tbl
+                    customTags: schematype=colocated,partition=range,pkey=composite,cardinality=multiple,projection=indexed,filtercount=one,filteron=rangeskey,filtertype=subquery,queryshape=join,joinon=rangepkey,jointables=three,joinconditions=two
+                    weight: 100
+                    queries:
+                        - query: select pkeyBigint1M_1.col_bigint_id_1, pkeyBigint1M_2.col_bigint_id_1, pkeyBigint1M_3.col_bigint_id_1 from pkeyBigint1M_1,pkeyBigint1M_2,pkeyBigint1M_3 where pkeyBigint1M_2.col_bigint_card2_1 IN (select distinct(col_bigint_card2_1) from pkeyBigint1M_1 where col_bigint_card1_1 in (1000500)) and pkeyBigint1M_1.col_bigint_id_1=pkeyBigint1M_2.col_bigint_id_1 and pkeyBigint1M_1.col_bigint_id_1=pkeyBigint1M_3.col_bigint_id_1
+
+            - workload: JOING9_5_cascade_join_3tbls_withfilter_onpkey
+              run:
+                  - name: join_on_pk_3tbl
+                    customTags: schematype=colocated,partition=range,pkey=composite,cardinality=multiple,projection=indexed,filtercount=one,filteron=rangepkey,filtertype=subquery,queryshape=join,joinon=rangepkey,jointables=three,joinconditions=two
```
Comment (sonalsagarwal) [inline config/yugabyte/regression_pipelines/join_workloads/yb_colocated/JOING9_cascadded_joins.yaml:118]: filtertype should be in

File: config/yugabyte/regression_pipelines/join_workloads/yb_colocated/JOING9_cascadded_joins.yaml [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+                    weight: 100
+                    queries:
+                        - query: select pkeyBigint1M_1.col_bigint_id_1, pkeyBigint1M_2.col_bigint_id_1, pkeyBigint1M_3.col_bigint_id_1 from pkeyBigint1M_1 JOIN pkeyBigint1M_2 on pkeyBigint1M_1.col_bigint_id_1=pkeyBigint1M_2.col_bigint_id_1  JOIN pkeyBigint1M_3 on pkeyBigint1M_2.col_bigint_id_1=pkeyBigint1M_3.col_bigint_id_1 where pkeyBigint1M_1.col_bigint_card1_1 in (1000500)
+
+            -   workload: JOING9_6_join_on_pkey_2ndpart_3tbls_withfilter_onskey
+                run:
+                    -   name: join_on_pk_3tbl
+                        customTags: schematype=colocated,partition=range,pkey=composite,cardinality=multiple,projection=indexed,filtercount=one,filteron=rangepkey,filtertype=subquery,queryshape=join,joinon=mixedpkey,jointables=three,joinconditions=two
```
Comment (sonalsagarwal) [inline config/yugabyte/regression_pipelines/join_workloads/yb_colocated/JOING9_cascadded_joins.yaml:126]: filtertype should be IN

File: config/yugabyte/regression_pipelines/aggregate_workloads/postgres/AGGRG10_aggregate_cardinality.yaml (hunk 1 above)
Comment (shantanugupta-yb) [reply config/yugabyte/regression_pipelines/aggregate_workloads/postgres/AGGRG10_aggregate_cardinality.yaml:77]: Removed it

**Review by sonalsagarwal — APPROVED** [review]
LGTM

**Pull request merged**